pkg/istio-agent/sds-agent.go

```diff
@@ -203,6 +203,15 @@ type SDSAgent struct {
 func NewSDSAgent(discAddr string, tlsRequired bool, pilotCertProvider, jwtPath, outputKeyCertToDir, clusterID string) *SDSAgent {
 	a := &SDSAgent{}
 
+	a.SDSAddress = "unix:" + LocalSDS
+	a.ClusterID = clusterID
+
+	// If a workload is using file mounted certs i,e, CA Provider is not set, we do not to have to initialize CA.
+	if !shouldProvisionCertificates() {
+		log.Info("Workload is using file mounted certificates. Skipping CA initialization")
```

Comment:
The logging statement here says "Skipping CA initialization". I am not sure what CA initializations this function is doing. Can you elaborate on what CA initializations are skipped?

Reply:
I meant skipping the CA related properties of SDS agent. I will change it to `Skipping setting up CA related configuration`